pokemongo_bot/cell_workers/sniper.py

```diff
@@ -406,12 +449,15 @@ def work(self):
         else:
             self.bot.sniper_disabled_global_warning = False
             targets = []
+            target = {}
```

Comment:
Noted. Will delete it off. I changed this files too many times and lost track what I did.